cli/ops/bench.rs

```diff
@@ -51,15 +52,17 @@ fn op_bench_get_origin(state: &mut OpState) -> String {
 #[derive(Clone)]
 struct PermissionsHolder(Uuid, PermissionsContainer);
 
-#[op2]
+#[op2(reentrant)]
 #[serde]
 pub fn op_pledge_test_permissions(
   state: &mut OpState,
   #[serde] args: ChildPermissionsArg,
+  #[stack_trace] stack: Option<Vec<JsStackFrame>>,
```

Comment:
It might be nice if we instead stored this internally in the op state then had the ability to register updates to it:

```rs
state.on_stack_update(|stack| permissions_container.set_current_stack(stack));
```

Then in the permissions container it could check what the current registred stack is. That way we don't need to pass this around so much.